lib/conditionals.md

```diff
@@ -5,8 +5,8 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A conditional uses the statements 'if', 'elsif' and 'else' to move the otherwise linear progression of a program in an alternative direction based on the true-ness or false-ness of the conditional statement. 
```

Comment:
I hope this is the piece I was missing:
Conditionals are also used with looping, where a program will repeat an action while a statement is true.